introspection-engine/introspection-engine-tests/tests/tables/mysql.rs

```diff
@@ -4,7 +4,7 @@ use introspection_engine_tests::test_api::*;
 use sql_introspection_connector::SqlIntrospectionConnector;
 use url::Url;
 
-#[test_connector(tags(Mysql))]
+#[test_connector(tags(Mysql), exclude(TiDB))]
```

Comment:
Not supported in TiDB? Does an issue exist?

Reply:
It seems that there is a compatibility problem here, the name of the generated unique key in TiDB is not consistent with MySQL.

```
Expect:
----
model Test {
  id       Int @id
  authorId Int @unique(map: "authorId") @default(autoincrement())
}

----

Actual:
----
model Test {
  id       Int @id
  authorId Int @unique(map: "authorid") @default(autoincrement())
}

----
```

I has created an issue [pingcap/tidb#39080](https://github.com/pingcap/tidb/issues/39080), and push the enginers to take a look on it.

Reply:
Ah, probably related to TiDB bein case-insensitive for column names by default. Interesting.

Reply:
This issue has been fixed by @wxbty. @wxbty, thank you for your work.

Reply:
Amazing. Do you have an issue or PR link? Just to connect the dots.

Reply:
issue: https://github.com/pingcap/tidb/issues/39080
fixed PR: https://github.com/pingcap/tidb/pull/39145

Reply:
The fixes for these compatibility-related issues will be released in the next release version (TiDB 6.4).

Reply:
Seems that one will make this PR a lot smoother and you will be able to fix quite a lot of things. Nice!

Reply:
😢 We may miss this release cycle, this fix PR may have to wait until TiDB 6.5. Version 6.4 has been frozen earlier.

Reply:
Shame. When will that be?

Is there a preview release or something? Happy to test with prerelease.

Reply:
We will publish the [Nightly](https://hub.docker.com/layers/pingcap/tidb/nightly/images/sha256-86ff12c369117872c0c3a3aafcf8b5d78efbf44b6debae6c5fa7fd81ca8bd1fb?context=explore) version in the Docker image.

The Nightly version can pass the test smoothly. I'm not sure whether you can accept the Nightly version in the prisma test.

According to the previous release frequency, TiDB will release a mirror version every one to two months.

https://docs.pingcap.com/tidb/dev/release-notes

Reply:
We can totally put the nightly image in the test here, as long as it is stable enough for us to learn what we want to learn - where Prisma and TiDB do not work together well right now.

Before merging this, we of course would love to switch to a different image then. (If we switch to nightly, I would open another PR that forks of the branch with the nightly to replace it with the current stable/minor so we can also monitor what _current_ users of TiDB would experience)

Reply:
Can you please open another PR into this branch that replaces the docker container with the nightly version and removes this test file modification @Mini256?